SU-HSDO/suhumsci
# Feat(STN-761):  Splash font in callout refinement

# READY FOR REVIEW

## Summary


Jira Card: https://sparkbox.atlassian.net/browse/STN-761

## Need Review By (Date)
8/20

## Urgency
medium

## Steps to Test
1. `npm run test`
2. `npm run build`
3. Add a Flexible Page with a Callout Box, and a Text Area underneath so the Callout Box will properly float beside the text area.
4. Add Splash Font text with the inline text styling on the callout box.
5. Verify that the callout box, splash font text no longer has large extra padding at the top of the splash font.
6. Add two separate lines of splash font text in the same callout box
7. Verify that the two splash font text still have proper spacing from each other.
8. Set the Callout Box to use a well background
9. Verify that the spacing is still accurate with the well on the callout box and splash font
10. Verify the spacing remains accurate with both Left and Right alignment
11. Verify the spacing doesn't break on mobile
12. Verify spacing on both themes still look correct.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
